app/services/like_service.rb

```diff
@@ -6,6 +6,6 @@ def create(params, user)
 
   def destroy(id, params, current_user)
     like = Like.find_by!(likable_id: id, user_id: current_user.id, likable_type: params[:likable_type])
-    Like.destroy!(like.id)
+    Like.destroy(like.id)
```

Comment:
Like.destroy(like.id) will make one more read to the database which is inefficient as a read has already been done in the line above. So please use

```ruby
like.destroy!
```

Reply:
like.destroy(like.id)! or just like.destroy! ?